### ISSUE TITLE
Add commands to view jobs and users and other things?

### ISSUE THREAD

**Issue by lars-t-hansen** [NAICNO/Jobanalyzer#32]
In order to develop and debug queries against the sonar data it will be necessary to look at data for specific jobs and users that trigger the queries.  I'm going to have this bug as a locus to discuss such inspection commands.

We can (somewhat) view the run-time behavior of a job over time with the `sonalyze load --job=...` command, perhaps this needs more options or more fields to be printed.

Most summary information about a job can be viewed with `sonalyze jobs --job=... --fmt=...` even though it's a little clunky, again maybe improve on its UI a little.

Most summary information about a user can be viewed with `sonalyze jobs --user=... --fmt=...`, again the same as above.

Most summary information about the sonar data itself can be viewed with `sonalyze jobs -v`.

**Comment by lars-t-hansen** [comment]
Also see #55 for a use case for `sonalyse user` to look at user's *aggregate* behavior across systems and jobs, something the `jobs` verb cannot do.

**Comment by lars-t-hansen** [comment]
Consider this analysis of a mysteriously failing genomics pipeline:
```
$ ./sonalyze jobs -u- -j1119125 --host ml9 -f3d --breakdown=command
level  jobm     user      duration  host  cpu-avg  cpu-peak  mem-avg  mem-peak  gpu-avg  gpu-peak  gpumem-avg  gpumem-peak  cmd
       1119125  pubuduss  0d18h40m  ml9   2782     3147      282      286       0        0         0           0            bwa,samtools
*      1119125  pubuduss  0d16h15m  ml9   3050     3137      6        6         0        0         0           0            bwa
*      1119125  pubuduss  0d18h30m  ml9   127      1203      280      281       0        0         0           0            samtools
```
along with this (abridged):
```
$ ./sonalyze load --job=1119125 --host ml9 -f4d --none 
HOST: ml9.hpc.uio.no
date        time   cpu   mem  gpu  gpumem
2023-10-21  08:35  3074  5    0    0
2023-10-21  08:40  3094  5    0    0
2023-10-21  08:45  3093  264  0    0
2023-10-21  08:50  3100  264  0    0
2023-10-21  08:55  3093  264  0    0
2023-10-21  09:00  3105  267  0    0
2023-10-21  09:05  3102  267  0    0
2023-10-21  09:10  3090  267  0    0
2023-10-21  09:15  3099  267  0    0
2023-10-21  09:20  3094  273  0    0
2023-10-21  09:25  3107  273  0    0
2023-10-21  09:30  3111  273  0    0
2023-10-21  09:35  3098  273  0    0
2023-10-21  09:40  3108  273  0    0
2023-10-21  09:45  3115  273  0    0
2023-10-21  09:50  3096  273  0    0
2023-10-21  09:55  3108  273  0    0
2023-10-21  10:00  3100  273  0    0
2023-10-21  10:05  3115  273  0    0
2023-10-21  10:10  3102  285  0    0
...
2023-10-22  00:50  3120  285  0    0
2023-10-22  00:55  97    280  0    0
...
2023-10-22  03:10  657   280  0    0
2023-10-22  03:15  658   280  0    0
```
These are interesting parts of the puzzle but not the full view.  What we would really like to see here is the load induced by each part of the pipeline, over time - essentially a --breakdown option for the load verb.

**Comment by lars-t-hansen** [comment]
From the above we can intuit that bwa started up, went up to about 5GB RAM almost immediately, and then ran for about 10 minutes before samtools suddenly kicked in, grabbed 256GB of RAM, but basically sat around and waited for data for a long time.  Then, just before 00:55Z, bwa was done and samtools jumped from 1 CPU to about 7, before failing suddenly a couple hours later (with a strange disk error).  @PubuduSaneth 

This also seems connected to #9, in which sonar can be run at a higher frequency to capture data with better granularity.

**Comment by lars-t-hansen** [comment]
Since #9 landed what's left here is mostly #55, so closing this one.
